test/stresstest/server.py

```diff
@@ -0,0 +1,58 @@
+########################################################################
+# Copyright (c) 2020 Robert Bosch GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+########################################################################
+
+import subprocess
+import os.path
+import config
+import signal
+
+class Server:
+    def handleSIGINT(self, signum, _):
+        if self.process != 0:
+            os.kill(self.process, signal.SIGTERM)
+    def build(self):
+        home = os.path.expanduser("~")
+        build_path = home + '/kuksa.val/kuksa-val-server/build'
+        isdir = os.path.isdir(build_path)
+        if not isdir:
+            subprocess.check_call(f"mkdir {build_path}", stdin=None, stdout=None, stderr=None)
+        print(os.getcwd())
+        src_path = build_path + '/src'
+        cmake_cmd = ["cmake",'..', build_path]
+        subprocess.check_call(cmake_cmd, stdin=None, stdout=None, stderr=None)
+        subprocess.check_call('make', cwd=src_path, stdin=None, stdout=None, stderr=None)
+    def start(self):
+        port = self.cfg.get('port')
+        address = self.cfg.get('ip')
+        home = os.path.expanduser("~")
+        build_path = home + '/kuksa.val/kuksa-val-server/build'
```

Comment:
As you are defining/using the same `build_path` within `build()` and `start()`, you could define a `self.build_path` in `__init__()` and use the same one everywhere I think.